css/main.css

```diff
@@ -524,7 +524,11 @@ a.btn:hover {
     width: 100%;
   }
 }
-
+@media (max-width: 658px) {
```

Comment:
Where does this number come from? As far as I see, this puts the two highlight boxes right next to eachother without space between in case the text wraps. However, with 668px width that does not happen due to the number above.

<img width="710" alt="screen shot 2017-09-01 at 09 41 30" src="https://user-images.githubusercontent.com/330324/29959929-e68ae258-8ef9-11e7-9d99-b8f11100bd86.png">

To be honest, I have no idea what branding guidelines say about this. In any case, I think this is not a very adaptive solution, as it highly depends on the length of the text. Any thoughts on this?

Reply:
> Where does this number come from?

I looked at the break point from where the text starts to get wrapped and break into two lines.
Ultimately I was just trying to replicate the view as it is on the current site.

> To be honest, I have no idea what branding guidelines say about this. In any case, I think this is not a very adaptive solution, as it highly depends on the length of the text. Any thoughts on this?

I agree with you that this is not an adaptive solution, I tried to give a quick fix based on current length of the text and without proper branding guidelines it's really hard to come up with a viable solution.